src/client/settings/Settings.js

```diff
@@ -27,6 +27,7 @@ import requiresLogin from '../auth/requiresLogin';
 import LANGUAGES from '../translations/languages';
 import { getLanguageText } from '../translations';
 import './Settings.less';
+import PackageJson from '../../../package.json';
```

Comment:
I would just call this variable `package`.

Reply:
Seems the word is reserved. I'll make the p lowercase.